PowerPointLabs/PowerPointLabs/SyncLab/ObjectFormats/LineArrowFormat.cs

```diff
@@ -1,12 +1,22 @@
-﻿using System.Drawing;
+﻿using System;
+using System.Drawing;
 using Microsoft.Office.Interop.PowerPoint;
+using Graphics = PowerPointLabs.Utils.Graphics;
 
 namespace PowerPointLabs.SyncLab.ObjectFormats
 {
     class LineArrowFormat
     {
         public static bool CanCopy(Shape formatShape)
         {
+            try
+            {
+                SyncFormat(formatShape, formatShape);
+            }
+            catch (Exception)
```

Comment:
This isn't meant to be an actual exception. There does not seem to be a way to check if a certain shape actually has an arrowhead format, than to access it. However if we access it and it isn't there, it throws an exception. This is why I am catching it here, then the function returns false, that there is no arrow :P